src/App.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import '../src/css/style.css';
-import Navbar from './pages/Navbar';
+import '../src/css/responsive.css';
+import Navbar from './components/Navbar';
+import CreateAccount from './components/CreateAccount';
 import LoginPage from './components/LoginPage';
+import ProfilePage from './pages/ProfilePage/ProfilePage';
 import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
 
+
 function App() {
     return (
         <Router>
           {/* <Navbar /> */}
           <Routes>
```

Comment:
use latest version of route

Reply:
As per NPM, latest version of react-router-dom is 6.10.0 we're using the same.